sql/schema.sql

```diff
@@ -46,4 +46,12 @@ CREATE TABLE IF NOT EXISTS sessions
 	refresh_token VARCHAR   NOT NULL,
 	id_token      VARCHAR   NOT NULL,
 	PRIMARY KEY (id)
+);
+
+CREATE TABLE IF NOT EXISTS api_tokens
```

Comment:
not sure if this should be a migration instead ? but migrations didn't seem loaded anywhere